internal/ethapi/backend.go

```diff
@@ -137,6 +137,11 @@ func GetAPIs(apiBackend Backend) []rpc.API {
 			Version:   "1.0",
 			Service:   NewPrivateAccountAPI(apiBackend, nonceLock),
 			Public:    false,
+		}, {
+			Namespace: "db",
+			Version:   "1.0",
+			Service:   NewPublicDbAPI(apiBackend),
+			Public:    true,
```

Comment:
Perhaps we should convert it to private API? Private APIs can also be exposed(http, ws) IIUC if users specify it, Public APIs will be exposed if the API module list specified by users is empty.